db/migrate/20170111135212_rename_draft_to_published.rb

```diff
@@ -0,0 +1,14 @@
+class RenameDraftToPublished < ActiveRecord::Migration
+  def change
+    reversible do |dir|
+      dir.up do
+        Event.all.each {|e| e.toggle!(:draft)}
+      end
+      dir.down do
+        Event.all.each {|e| e.toggle!(:published)}
```

Comment:
I think it should be :draft here too. When i run the migration it works, but when I try a rollback it fails with the error that Event has no published attribute. I guess the rename happens first